WooCommerce/Classes/ViewRelated/Products/Add Product/AddProductFromImage/AddProductFromImageViewModel.swift

```diff
@@ -161,6 +177,23 @@ private extension AddProductFromImageViewModel {
             })
         }
     }
+
+    /// Disables regenerate button if there is at least one selected and non-empty text.
```

Comment:
nit: I think this should read as `Enables regenerate button if there is at least one selected and non-empty text.`.

Reply:
Updated in 08d78ba8417e1d79d27173e067c2af75d6439793.